crmsh/ui_sbd.py

```diff
@@ -0,0 +1,538 @@
+            return False
+
+    @command.completers_repeating(sbd_configure_completer)
+    def do_configure(self, context, *args) -> bool:
+        '''
+        Implement sbd configure command
+        '''
+        try:
+            for service in (constants.PCMK_SERVICE, constants.SBD_SERVICE):
+                if not self.service_is_active(service):
+                    return False
+            if not args:
+                raise self.SyntaxError("No argument")
+
+            if args[0] == "show":
+                self._configure_show(args)
+                return True
+            parameter_dict = self._parse_args(args)
+            if sbd.SBDUtils.is_using_disk_based_sbd():
+                self._configure_diskbase(parameter_dict)
+            elif sbd.SBDUtils.is_using_diskless_sbd():
+                self._configure_diskless(parameter_dict)
+            return True
+
+        except self.SyntaxError as e:
+            logger.error('%s', str(e))
+            usage = self.configure_usage
+            if usage:
+                print(usage)
+            return False
+
+    def do_purge(self, context) -> bool:
+        '''
+        Implement sbd purge command
+        '''
+        if not self.service_is_active(constants.SBD_SERVICE):
+            return False
+        sbd.purge_sbd_from_cluster()
+        sbd.SBDManager.restart_cluster_if_possible()
+        return True
+
+    def _print_sbd_type(self):
+        if not self.service_manager.service_is_active(constants.SBD_SERVICE):
+            return
+        print("# Type of SBD:")
+        if self.device_list_from_config:
+            print("Disk-based SBD configured")
+        else:
+            print("Diskless SBD configured")
+        print()
+
+    def _print_sbd_status(self):
+        padding = 2
+        status_len = 8
+        max_node_len = max(len(node) for node in self.cluster_nodes) + padding
+
+        print(f"# Status of {constants.SBD_SERVICE}:")
+        print(f"{'Node':<{max_node_len}}|{'Active':<{status_len}}|{'Enabled':<{status_len}}|Since")
+        for node in self.cluster_nodes:
+            is_active = self.service_manager.service_is_active(constants.SBD_SERVICE, node)
+            is_active_str = "YES" if is_active else "NO"
+            is_enabled = self.service_manager.service_is_enabled(constants.SBD_SERVICE, node)
+            is_enabled_str = "YES" if is_enabled else "NO"
+            systemd_property = "ActiveEnterTimestamp" if is_active else "ActiveExitTimestamp"
+            since_str_prefix = "active since" if is_active else "disactive since"
+            systemctl_show_cmd = f"systemctl show {constants.SBD_SERVICE} --property={systemd_property} --value"
+            since = self.cluster_shell.get_stdout_or_raise_error(systemctl_show_cmd, node) or "N/A"
+            print(f"{node:<{max_node_len}}|{is_active_str:<{status_len}}|{is_enabled_str:<{status_len}}|{since_str_prefix}: {since}")
+        print()
+
+    def _print_watchdog_info(self):
```

Comment:
Changed as: (when cluster service is stopped at alp-1)
```
alp-1:~ # crm sbd status
# Status of sbd.service:
Node   |Active  |Enabled |Since
alp-1  |NO      |YES     |disactive since: Fri 2024-11-29 21:34:44 CST
alp-2  |YES     |YES     |active since: Fri 2024-11-29 21:32:01 CST

# Status of sbd process on alp-1:
No sbd process found

# Status of sbd process on alp-2:
├─8993 sbd: watcher: /dev/sda5 - slot: 1 - uuid: 47509131-0eaa-4c6f-80ed-721a6aaf028e

# Watchdog info:
Node   |Device          |Driver    |Kernel Timeout
alp-1  |N/A             |N/A       |N/A
alp-2  |/dev/watchdog0  |iTCO_wdt  |10

# Status of fence_sbd:
resource stonith-sbd is running on: alp-2
```